data/fixtures/recorded/surroundingPair/parseTree/typescript/changePair.yml

```diff
@@ -12,9 +12,9 @@ command:
   usePrePhraseSnapshot: true
 initialState:
   documentContents: |-
-    (
-    // )
-    )
+    {
```

Comment:
The old example is not valid typescript and gives an error node. This pull request changes so we actually check if the text fragment is erroring or not.